Config for opendebatequestions.com

jbradberry opened the pull request:
OP-171

jbradberry committed:
Added the necessary entries in fabulaws-config.yml to stand up the new site.

jbradberry committed:
Added the staging01 task to fabfile.py.

jbradberry committed:
Doubled the sizes of the instances for prod01

To match what has been done for presidential.  Excepting, of course, the cache instance which is already 4x what florida had.

tobiasmcnulty commented on fabfile.py:
```diff
@@ -25,6 +25,11 @@ def presidential(deployment_tag=env.default_deployment, answer=None):
 
 
 @task
+def staging01(deployment_tag=env.default_deployment, answer=None):
```
need prod01 as well?

jbradberry replied:
Added.

tobiasmcnulty commented on fabulaws-config.yml:
```diff
@@ -181,6 +202,12 @@
       db-slave: m4.4xlarge
       web: c3.2xlarge # for faster AMI creation (no swap file)
       worker: m4.xlarge
+    prod01:
```
I'd vote for the original prod sizes (half this all around), if not half that. Current prod is quite over-provisioned, and I believe they expect less traffic on this site.

tobiasmcnulty replied:
e.g.:

```
  cache: c4.large
  db-master: m4.xlarge
  db-slave: m4.xlarge
  web: c3.large # for faster AMI creation (no swap file)
  worker: m4.large
```

jbradberry replied:
Updated.

tobiasmcnulty commented on fabulaws-config.yml:
```diff
@@ -232,6 +271,12 @@
       web: 10 # 10 GB data (no swap)
       worker: 66 # 50 GB data + 16 GB swap
+    prod01:
+      cache: 25 # 10 GB data + 15 GB swap
```
Just a note that these could be adjusted along with the instance sizes, as the need for swap is reduced. Deciding not to worry about it would be a perfectly reasonable decision IMO, too (unless of course the instance sizes need to increase further at any point, in which case the server creation will almost certainly break if the EBS volumes aren't increased in size).

jbradberry committed:
Add a prod01 fabric task.

jbradberry committed:
Adjust the size of the instances for this environment.

The presidential site wound up being over-provisioned, so let's not
make that mistake again.

vkurup commented on fabulaws-config.yml:
```diff
@@ -2,6 +2,8 @@
   # "staging": staging
```
should we update this line to make it clear that it's staging.presidentialopenquestions?

jbradberry replied:
Good point.

vkurup commented on fabulaws-config.yml:
```diff
@@ -55,14 +57,17 @@
       staging: develop
       florida: master
       presidential: develop # XXX change to master for final prod deploy 9/14/16
```
Is the above comment relevant anymore?

vkurup commented:
A couple comments (about comments), but looks good to :shipit:, especially since you're proven it's working.

jbradberry committed:
Address the final review comments

- remove the comment about using the master branch, which we're in no hurry to do
- note that the staging environment is staging.presidentialopenquestions
